queries.sql

```diff
@@ -23,3 +24,67 @@ SELECT * FROM animals WHERE name!='Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--Phase 2 update animals table
+
+--Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made.
+--Then roll back the change and verify that species columns went back to the state before transaction
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+-- Transactions to all animals with names ending with mon
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+--Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+BEGIN;
```

Comment:
- Kindly remove the `BEGIN` statement here as the transaction already began on line 40. Here 👇  is what the requirement says

![](https://i.imgur.com/6PNLHga.png)